sdk/core/Azure.Core.TestFramework/src/TestProxy.cs

```diff
@@ -67,6 +67,7 @@ private TestProxy(string proxyPath, bool debugMode = false)
                 UseShellExecute = false,
                 RedirectStandardOutput = true,
                 RedirectStandardError = true,
+                WorkingDirectory = TestEnvironment.RepositoryRoot,
```

Comment:
It doesn't seem like this approach is working and it would also break our current debug workflow. Could we pass the context directory to the test proxy instead?

Reply:
Really? Yes, we can. I'll adjust the `arguments` to pass `--storage-location`.

Reply:
Done.